spec/compressor_spec.rb

```diff
@@ -546,6 +546,22 @@
         },
       ],
     },
+    { title: 'D.6.a.  Response Examples with Huffman - nginx 1.14-1.15.2 responses',
+      type: :response,
+      table_size: 4096,
+      huffman: :always,
+      bypass_encoder: true,
+      streams: [
+        { wire: '2088 7689 aa63 55e5 80ae 16d7 17',
+          emitted: [
+            [':status', '200'],
+            ['server', 'nginx/1.15.2'],
+          ],
+          table: [],
+          table_size: 0,
```

Comment:
As it, above wire bytes are emitting a dynamic resize to 0 table entries?

Reply:
it does appear that's what nginx was doing:
[signal 0-byte HPACK's dynamic table size](https://trac.nginx.org/nginx/changeset/12cadc4669a7/nginx)

Reply:
Wait, what... 

> "NGINX doesn't use HPACK's dynamic table when encoding headers"

Wow. That's a disappointment of the day — ugh.

--- 

With respect to landing this patch. Let's update the spec description to be explicit what we're testing for, and otherwise we should be good.